tests/integration/v4/acl.v4.feature

```diff
@@ -0,0 +1,165 @@
+@acl
```

Comment:
this is similar to our acl.feature with minimal changes and some improvement on what we check and of course hydrated entities